### DISCUSSION TITLE
Add type hints ?

### DISCUSSION THREAD

**Discussion by Jota-Ve** [shapely/shapely#1316] [Ideas]
Hello, I'm new to GitHub and I'm trying to find something to contribute on. I've discovered this library recently and noticed that, although it seems to have good doc strings, it doesn't have actual type hints. Should I try to add them ?

**Comment by mwtoews** [comment]
See #721, which (while closed) welcomes Python 3.6+ annotations for shapely 2. Help would be appreciated.

**Reply by Jota-Ve** [reply]
Ok. I'm sorry but I've looked into it, but didn't find where I can install this 2.0 version. Can you provide me the link ?

**Reply by mwtoews** [reply]
The current `main` branch is development towards shapely 2.
